snippets/csharp/VS_Snippets_CLR_Classic/classic NotifyFilters Example/CS/source.cs

```diff
@@ -5,62 +5,59 @@
 
 public class Watcher
 {
-
     public static void Main()
     {
-    Run();
-
+        Run();
     }
 
-    [PermissionSet(SecurityAction.Demand, Name="FullTrust")]
+    [PermissionSet(SecurityAction.Demand, Name = "FullTrust")]
     public static void Run()
     {
-        string[] args = System.Environment.GetCommandLineArgs();
+        string[] args = Environment.GetCommandLineArgs();
 
         // If a directory is not specified, exit program.
-        if(args.Length != 2)
+        if (args.Length != 2)
         {
             // Display the proper way to call the program.
             Console.WriteLine("Usage: Watcher.exe (directory)");
             return;
         }
 
         // Create a new FileSystemWatcher and set its properties.
-        FileSystemWatcher watcher = new FileSystemWatcher();
-        watcher.Path = args[1];
-        /* Watch for changes in LastAccess and LastWrite times, and
-           the renaming of files or directories. */
-        watcher.NotifyFilter = NotifyFilters.LastAccess | NotifyFilters.LastWrite
-           | NotifyFilters.FileName | NotifyFilters.DirectoryName;
-        // Only watch text files.
-        watcher.Filter = "*.txt";
+        using (FileSystemWatcher watcher = new FileSystemWatcher())
```

Comment:
The corresponding VB example is ~\samples\snippets\visualbasic\VS_Snippets_CLR_Classic\classic NotifyFilters Example\VB\source.vb